Generators/src/main/java/io/deephaven/gen/JavaFunction.java

```diff
@@ -147,13 +151,102 @@ public Class<?> getReturnClass() {
         }
 
         try {
-            return TypeUtils.getErasedType(returnType);
+            return getErasedType(returnType);
         } catch (UnsupportedOperationException e) {
             log.warning("Unable to determine Class from returnType=" + returnType.getTypeName());
             return null;
         }
     }
 
+    /**
+     * Determine the Class from the Type.
+     */
+    private static Class<?> getErasedType(Type paramType) {
+        if (paramType instanceof Class) {
+            return (Class<?>) paramType;
+        } else if (paramType instanceof ParameterizedType) {
+            return (Class<?>) // We are asking the parameterized type for its raw type, which is always Class
```

Comment:
Not actionable: I appreciate that you took the time to correct `"it's"` to `"its"`.